tools/ClassGen/NodeGen.cpp

```diff
@@ -1259,6 +1259,28 @@ int main(int argc, char **argv) {
                     "V4 by outputting indices and scalar tensor with number of "
                     "valid indices. It pads the rest with global MIN box.");
 
+  //===--------------------------------------------------------------------===//
+  //                Region of Interest nodes
+  //===--------------------------------------------------------------------===//
+
+  BB.newNode("ROIAlign")
+      .addInput("FeatureMap")
+      .addInput("Boxes")
+      .addInput("BatchIndices")
+      .addMember(MemberType::String, "Mode")
+      .addMember(MemberType::Unsigned, "OutputHeight")
+      .addMember(MemberType::Unsigned, "OutputWidth")
+      .addMember(MemberType::Unsigned, "SamplingRatio")
+      .addMember(MemberType::Float, "SpatialScale")
+      .addResultFromCtorArg()
+      .setDocstring(
+          "Given FeatureMap tensor of [N,H,W,C], Boxes tensor of [K,4],"
```

Comment:
[N,H,W,C] is the standard layout in Glow , but I see the implementation above is in [N,C,H,W] , right?

Reply:
Modified to NHWC